docs/animation/implementing.md

```diff
@@ -0,0 +1,103 @@
+Implementing the API
+======================
+
+Depending on what you want to animate with the API, code-side implementation is a bit different.
+Documentation on the ASM api itself (for controlling the animation) is found on the [ASM][asm] page because it is independent of what
+you are animating.
+
+Blocks
+--------
+
+Animations for blocks are done with the AnimationTESR, which is a FastTESR. Because of this, having a TileEntity for your block
+is necessary. Your TileEntity must provide the `ANIMATION_CAPABILITY`, which is recieved by calling its `.cast` method with your
+ASM. Your block must also render in the `ENTITYBLOCK_ANIMATED` render layer if you do not provide a `StaticProperty` in the block's blockstate.
+
+The `StaticProperty` is a property you can add to your block's blockstate by adding `Properties.StaticProperty` to the list of your block's properties inside
```

Comment:
Although it's kind of obvious from the name, you should mention that all parts of the model that *can* be static should probably also be rendered in the static state, that's the whole point.